cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_log_files.json

```diff
@@ -522,6 +522,28 @@
         "HeadNode"
       ],
       "feature_conditions": []
+    },
+    {
+      "timestamp_format_key": "default",
+      "file_path": "/var/log/parallelcluster/slurm_prolog_epilog.log",
```

Comment:
I think we're missing to add these files in the CloudWatch dashboard (and related unit tests).